Core/Controller/EditTransferenciaStock.php

```diff
@@ -54,6 +57,77 @@ protected function createViews()
         $this->addEditListView('EditLineaTransferenciaStock', 'LineaTransferenciaStock', 'lines-transfer');
     }
 
+    /**
+     * Run the controller before actions
+     *
+     * @param string $action
+     */
+    protected function execPreviousAction($action)
```

Comment:
the work does not fit the description. The work had to be done in the model, not in the controller